lib/ioc-rc

```diff
@@ -555,7 +557,8 @@ __rc_jails () {
         local boot_order=$(sort -n $boot_list)
         local shutdown_order=$(sort -rn $boot_list)
     else
-        __info "no jails found with property boot=on, exiting.."
+        __info "no jails found with property boot=on or
+        with jail_hostid: $hostid on this host, exiting.."
```

Comment:
Not quite :) I'll fix it up, so no worries. As for the commits, do a `git rebase -i a65ce3d` and then:

Squash `fe8dfd7` and `3722b48` onto `34d3b0c`
Squash `dd13723` and `f7596a5` onto 'a65ce3d'.